source/Calamari.Shared/Integration/Packages/Download/S3PackageDownloader.cs

```diff
@@ -82,46 +85,44 @@ public PackagePhysicalFileMetadata DownloadPackage(string packageId,
                 return downloaded;
             }
 
-            int retry = 0;
+            var retry = 0;
             for (; retry < maxDownloadAttempts; ++retry)
             {
                 try
                 {
-                    log.Verbose($"Attempting download of package {packageId} version {version} from S3 bucket {bucketName}. Attempt #{retry + 1}");
+                    var region = variables.Get(AuthenticationVariables.Aws.Region) ?? GetBucketsRegion(feedUsername, feedPassword, bucketName);
```

Comment:
This variable gets set in Octopus if the feed is an S3 bucket feed

Reply:
Reviewing with whitespace hidden in GH UI makes this much easier